### PR TITLE
Sync with internal changes.

### PR DESCRIPTION
**Pull request by mingxingtan**
PiperOrigin-RevId: 337767572

### CODE REVIEW

File: efficientdet/aug/gridmask.py [hunk 1]
```diff
@@ -12,169 +12,125 @@
...
+        tf.cast(
+            (self.gridmask_size_ratio + 1), tf.float32) * tf.math.maximum(h, w),
+        tf.int32)
+    self.mask_w = mask_w
+    mask = tf.zeros(shape=[mask_h, mask_w], dtype=tf.int32)
+    gridblock = tf.random.uniform(
+        shape=[],
+        minval=int(tf.math.minimum(h * 0.5, w * 0.3)),
+        maxval=int(tf.math.maximum(h * 0.5, w * 0.3)) + 1,
+        dtype=tf.int32)
```
Comment (mingxingtan) [inline efficientdet/aug/gridmask.py:80]: tf.random.uniform fails if minval==maxval, so I add 1 for maxval. Same for other tf.random in this file.

Comment (kartik4949) [reply efficientdet/aug/gridmask.py:80]: right!

**Comment by kartik4949** [comment]
LGTM!

**Review by kartik4949 — APPROVED** [review]

**Comment by fsx950223** [comment]
Merge tflite_path and quantization_type to one option which is similar to tensorrt.

**Comment by kartik4949** [comment]
@mingxingtan there is a TODO in hparams file for num_classes change (Line 191 )

File: efficientdet/keras/inspector.py [hunk 2]
```diff
@@ -60,6 +61,9 @@
 flags.DEFINE_string('saved_model_dir', '/tmp/saved_model',
```
Comment (fsx950223) [inline efficientdet/keras/inspector.py:61]: '/tmp/saved_model' -> None

**Comment by mingxingtan** [comment]
> Merge tflite_path and quantization_type to one option which is similar to tensorrt.

Good point. Done

**Pull request merged**